src/objects/core/models.py

```diff
@@ -41,8 +42,14 @@ def url(self):
         # zds_client.get_operation_url() can be used here but it increases HTTP overhead
         return f"{self.service.api_root}objecttypes/{self.uuid}"
 
-    def clean(self):
+    def clean_fields(self, exclude: Optional[Iterable[str]] = None) -> None:
```

Comment:
Minor, but we could use this syntax:

```suggestion
    def clean_fields(self, exclude: Iterable[str] | None = None) -> None:
```